Makefile

```diff
@@ -14,6 +14,12 @@ export GITHUB_WORKSPACE = $(shell pwd)
 run:
 	@bash ./entrypoint.sh $(DEFAULT_VERSION) $(DEFAULT_COMMAND) $(TEST_FILEPATH) $(DEFAULT_TEMPLATE) $(DEFAULT_LANGUAGE) $(DEFAULT_OUTPUT) $(DEFAULT_PARAMETERS) $(DEFAULT_CUSTOM_COMMANDS) 
 
+# Updates the action.yml file with the latest version of the action got from $VERSION env variable
+generate-assets:
+	version=$$(echo ${VERSION} | sed 's/v//g'); \
+	sed "s/docker:\/\/asyncapi\/github-action-for-cli:.*/docker:\/\/asyncapi\/github-action-for-cli:$$version'/g" action.yml > action.yml.tmp
+	@mv action.yml.tmp action.yml
```

Comment:
it is a bit hack to run bump script as part of tests

can't we run it through npm scripts maybe?

Reply:
No these are not run as part of tests. All commands in make-file are separate script.